### PR TITLE
Desktop version of my portfolio

### PR DESCRIPTION
**Pull request by axcell-0**
"to creat my desktop version I used "
-Media queries
-Github flows
-Index.html files
-App.css files

### CODE REVIEW

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @axcell-0, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
- [ ] Kindly consider watching the video below for more details. And remember to stick to the design template as much as possible.
[Watch](https://user-images.githubusercontent.com/107192076/230689825-14a7f584-afdf-44f6-a834-3d22f2efcbfd.mp4)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/styles/app.css [hunk 1]
```diff
@@ -445,3 +469,216 @@ textarea {
...
+    margin-top: 50px;
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: auto;
+    grid-column-gap: 32px;
+    grid-row-gap: 24px;
+    padding-right: 100px;
+    padding-left: 100px;
+  }
```
[the remark this answers is not in the document]
Comment (axcell-0) [reply assets/styles/app.css:526]: @Strangeal thanks very much for the review I am very sorry I asked for a secound opinion which I was not supposed to do that the instruction was clear nut I didn't understand, it's when I watch the video before I understood now I don't know how to remove it. thanks again I will do exactly as you said. happy coding

**Comment by axcell-0** [comment]
@Strangeal thanks for the review the only thing lacking now is how to make the third card to come in the middle so please can you help me on how to figure it out thanks alot

**Review by amon-cofie — APPROVED** [review]
Hi @axcell-0 , 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You have a descriptive pull request summary
❌  Your commit messages are not imperatives and not descriptive
🚀 You used GitHub flow properly
🚀 You have no linter errors


## Optional suggestions

- [ ] [OPTIONAL] Great job on fixing the problems that the previous reviewer pointed out 👍. However, on screens of size 768px and  I found the following issues and I think fixing them will improve upon the UI/UX of the page: 

> Your ```about``` text seems to clipped off by the edges of the screen. Perhaps its too wide
> There seems to be a few margin issues to the right of the project cards in your works section.

image
 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by amon-cofie** [comment]
> @Strangeal thanks for the review the only thing lacking now is how to make the third card to come in the middle so please can you help me on how to figure it out thanks alot

If you want to perfectly center a container horizontally, one thing you can do is to set both margin-right and margin-left to auto. Maybe this can help you

**Review by nicupop729 — COMMENTED** [review]
Hello @axcell-0  👋,

_This is an additional review as a result of the second opinion request._

Kudos for having your project approved! :clap:

When asking for a Second Opinion review, please wait to receive it before asking for another review in the dashboard.

Keep rocking 🚀!

**Pull request merged**